harness/harness-common.rb

```diff
@@ -28,3 +28,28 @@ def use_gemfile(extra_setup_cmd: nil)
   # Need to be in the appropriate directory for this...
   require "bundler/setup"
 end
+
+def get_rss
```

Comment:
Needs a one-sentence comment to explain what RSS is for those not in the know.

Reply:
Added a comment with those things.